sys/include/net/sock/ip.h

```diff
@@ -316,9 +316,10 @@ typedef struct sock_ip sock_ip_t;
  *          elsewhere
  * @return  -EAFNOSUPPORT, if `local != NULL` or `remote != NULL` and
  *          sock_ip_ep_t::family of @p local or @p remote is not supported.
- * @return  -EINVAL, if `proto` is not supported or if sock_ip_ep_t::netif of
- *          @p local or @p remote is not a valid interface or contradict each
- *          other (i.e. `(local->netif != remote->netif) &&
+ * @return  -EINVAL, if sock_ip_ep_t::addr of @p remote is an invalid address.
+ * @return  -EINVAL, if sock_ip_ep_t::netif of @p local or @p remote is not a
+ *          valid interface or contradict each other
```

Comment:
Mh though in the first half of the sentence I use "is". Will find something better.